packages/cozy-stack-client/src/SettingsCollection.js

```diff
@@ -0,0 +1,36 @@
+import DocumentCollection from './DocumentCollection'
+import { dontThrowNotFoundError } from './Collection'
+
+export const SETTINGS_DOCTYPE = 'io.cozy.settings'
+
+/**
+ * Implements `DocumentCollection` API to interact with the /settings endpoint of the stack
+ */
+class SettingsCollection extends DocumentCollection {
+  constructor(stackClient) {
+    super(SETTINGS_DOCTYPE, stackClient)
+  }
+
+  /**
+   * async get - Calls a route on the /settings API
+   *
+   * @param  {string} path The setting route to call, eg `instance` or `context`
+   * @returns {object} The response from the route
+   */
+  async get(path) {
+    try {
+      const resp = await this.stackClient.fetchJSON('GET', `/settings/${path}`)
+
+      return DocumentCollection.normalizeDoctypeJsonApi(SETTINGS_DOCTYPE)(
+        resp.data,
+        resp
+      )
+    } catch (error) {
+      return dontThrowNotFoundError(error)
+    }
```

Comment:
can you test the `dontThrow`  ? :)

Reply:
Actually I took that from an existing file but it makes no sense in this context, we should pass down errors. I added a test for that.